pkg/cri/server/fake_container.go

```diff
@@ -0,0 +1,68 @@
+package server
```

Comment:
Updating container resources is out of scope of this PR. And since the integration tests all use [integration/remote/remote_runtime.go](https://github.com/containerd/containerd/blob/main/integration/remote/remote_runtime.go), I am going to add

```
if status.Resources.Windows == nil && status.Resources.Linux == nil {
		return fmt.Errorf("Neither Resources.Windows or Resources.Linux is not set")
	}
```

to [verifyContainerStatus](https://github.com/containerd/containerd/blob/e036bdc9857e1497bce729533b1955953207090e/integration/remote/utils.go#L80).

WDYT?

Reply:
> Updating container resources is out of scope of this PR.

We already have an UpdateContainerResources api.. thus making said updates in scope of this PR as the changes made in update should/will be reflected in the ContainerStatus response..

> since the integration tests all use [integration/remote/remote_runtime.go] (https://github.com/containerd/containerd/blob/main/integration/remote/remote_runtime.go), I am going to add

>if status.Resources.Windows == nil && status.Resources.Linux == nil {
>		return fmt.Errorf("Neither Resources.Windows or Resources.Linux is not set")
>	}
>to [verifyContainerStatus](https://github.com/containerd/containerd/blob/e036bdc9857e1497bce729533b1955953207090e/integration/remote/utils.go#L80).


I would prefer to not introduce this as a new requirement.